_layouts/pattern.html

```diff
@@ -11,41 +11,52 @@
   <a href="#" class="carousel-control next">Next</a>
   {% if page.images %}
     {% case page.images.size %}
-      {% when 1 %}
+      {% when 2 %}
       <div class="{% if page.illustration_version == 1%}large-12 medium-12{% else %}large-4 large-offset-4 medium-4 medium-offset-4{% endif %} small-12 cell">
-        <img src="{{ page.images[0].url }}" />
+        <img id="content" tabindex="0" src="{{ page.images[0].url }}" alt="{{ page.images[1].alt }}"/>
       </div>
-      {% when 2 %}
+      {% when 3 %}
       {% assign count = 0 %}
       {% for image in page.images %}
         {% if count == 0 %}
           <div class="large-4 large-offset-2 medium-4 medium-offset-2 small-12 cell">
         {% else %}
           <div class="large-4 medium-4 small-12 cell">
         {% endif %}
-            <img src="{{ image.url }}" />
+            <img id="content" tabindex="0" src="{{ image.url }}" alt="{{ image.alt }}" />
           </div>
         {% assign count = count | plus: 1 %}
       {% endfor %}
-      {% when 3 %}
+      {% when 4 %}
       <!-- When no script, multiple images and mobile screen size, display only 1 image. -->
       <noscript>
         <div class="no-script-image-div grid-x">
           <div class="three-image small-12 cell phone">
-            <img src="{{ page.images[1].url }}" alt="">
+            <img id="content" tabindex="0" src="{{ page.images[1].url }}" alt="{{ page.images[3].alt }}"/>
           </div>
         {% for image in page.images %}
           <div class="three-image desktop large-4 medium-4 small-4 cell">
-            <img src="{{ image.url }}" />
+            <!-- Hard-coded alt text as there is currently only one pattern with three images. Change this when new patterns added. -->
```

Comment:
The set of conditionals for accommodating the three image view is quite hard to wrap my head around. On the next iteration it would be good to think about how this could be refactored.